src/workflow/task_executor.py

```
@@ -0,0 +1,227 @@
+            
+            file_data = slack_api.get_image_from_slack(image_url)
+            if not file_data:
+                # 직접 다운로드 시도
+                try:
+                    import requests
+                    response = requests.get(image_url)
+                    if response.status_code == 200:
+                        file_data = response.content
+                except Exception as e:
+                    logger.log_error("이미지 다운로드 중 오류 발생", e)
+                    raise Exception("이미지를 다운로드할 수 없습니다.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)